### PR TITLE
chore: upgrade docs

### PR DESCRIPTION
**Pull request by Tahul**
Hey 😄

I finally got the `docs/` to work with latest Docus version.

Here is the PR updating documentation structure!

Feel free to comment if anything is missing/needed.

👋

### CODE REVIEW

File: docs/nuxt.config.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 
 export default withDocus({
-  docus: {
-    colors: {
-      primary: '#00DC82'
+  rootDir: __dirname,
+  socialImage: {
+    baseUrl: 'DOMIN',
+    chrome: {
+      browserWSEndpoint: 'wss://chrome.browserless.io/'
```
Comment (pi0) [inline docs/nuxt.config.js:8]: Is there any reason why docus is not setting this for default? (PS not sure but `nuxt-extend` should set `rootDir` from top layer if not we can investigate)

Comment (pi0) [reply docs/nuxt.config.js:8]: @Tahul

Comment (Atinux) [reply docs/nuxt.config.js:8]: Agree this should be set by default

Comment (Tahul) [reply docs/nuxt.config.js:8]: Resolved by https://github.com/nuxtlabs/docus/commit/bee93deced36a04f5b3471e6ffa37352400cccaf

File: docs/package.json [hunk 2]
```diff
@@ -1,15 +1,13 @@
 {
   "private": true,
   "scripts": {
-    "generate": "nuxt generate",
+    "generate": "nuxt generate --force-build",
```
Comment (pi0) [inline docs/package.json:4]: Is this to detect changes?

Comment (Tahul) [reply docs/package.json]: It was a remainder from Docus docs, it's fixed in last commit. :)

**Review by pi0 — APPROVED** [review]
LGTM

**Pull request merged**